<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGBLampWidget</class>
 <widget class="QWidget" name="RGBLampWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item alignment="">
      <widget class="QLabel" name="RGBLamp_deviceIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/static/icons/rgb-light-bulb.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorPickerButton">
       <property name="text">
        <string>Color Picker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toggleButton">
       <property name="text">
        <string>Toggle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightnessSlider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Status: OFF</string>
       </property>
      </widget>
     </item>
     <item alignment="">
      <widget class="QLabel" name="IPAddressLabel">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
